uni/lib/Makefile

```diff
@@ -33,6 +33,31 @@ selection3D.u: ../3d/selection3D.icn
 	$(UNICON) $(UFLAGS) -c selection3D.icn
 	rm selection3D.icn
 
+utf8.u: ../utf8/utf8.icn classobject.u
+	cp ../utf8/utf8.icn .
```

Comment:
In terms of an alternative name to library, I don't have a problem with just putting the four files into the lib directory and updating the Makefile to just compile these appropriately. This would make my life considerably easier in terms of the next set of packages that I want to include (still a work in progress) and I would not have to worry about the extra directory as I can place everything into lib locally. 

I will make the changes accordingly here and upload a commit to fix this.